portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,16 +63,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
magic constants

Reply:
I am not entirely sure what this comment is referring to.

Reply:
magic constants refers to the hardcoded strings. They're often referred to as magic since hardcoded values are sometimes hard to understand from someone other than the author's point of view.

Reply:
I understand what a magic constant is, but in the line of code this comment appears to be attached to there doesn't seem to be any magic constants.

Reply:
I was referring to all the constants in the function. "Comment" etc.

Reply:
I was referring to all the constant strings used in the function below. "Comment", "name-entry", "comment-entry" etc.